File: net/import-rich-text/CreateSimpleType.cs
// DocSection: import_rich_create_simple_type
// Tip: Find more about .NET SDKs at https://docs.kontent.ai/net
using Kentico.Kontent.Management;

var client = new ManagementClient(new ManagementOptions
{
    ApiKey = "<YOUR_API_KEY>",
    ProjectId = "<YOUR_PROJECT_ID>"
});

var response = await client.CreateContentTypeAsync(new ContentTypeCreateModel
{
    Name = "Simple Rich Text",
    Codename = "simple-rich-text",
    Elements = new ElementMetadataBase[]
    {
        new RichTextElementMetadataModel
        {
            Name = "Rich Text",
            ExternalId = "rich-text",
        },
    }
});
// EndDocSection

File: js/import-rich-text/CreateSimpleType.js
// DocSection: import_rich_create_simple_type
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
// Using ES6 syntax
import { ManagementClient, ElementModels } from '@kentico/kontent-management';

const client = new ManagementClient({
  projectId: '<YOUR_PROJECT_ID>',
  apiKey: '<YOUR_API_KEY>'
});

const response = await client.addContentType()
  .withData(
    {
      external_id: "simple-rich-text",
      name: "Simple Rich Text",
      elements: [
        {
          name: "Rich Text",
          type: ElementModels.ElementType.RichText,
          external_id: "rich-text"
        }
      ]
    }
  )
  .toPromise();
// EndDocSection